orchagent/bufferorch.cpp

```diff
@@ -218,29 +228,60 @@ void BufferOrch::generateBufferPoolWatermarkCounterIdList(void)
     }
 
     // Detokenize the SAI watermark stats to a string, separated by comma
-    string statList;
+    string statListPoolOnly;
     for (const auto &it : bufferPoolWatermarkStatIds)
     {
-        statList += (sai_serialize_buffer_pool_stat(it) + list_item_delimiter);
+        statListPoolOnly += (sai_serialize_buffer_pool_stat(it) + list_item_delimiter);
     }
-    if (!statList.empty())
+    string statListPoolAndShp = statListPoolOnly;
+    for (const auto &it : bufferSharedHeadroomPoolWatermarkStatIds)
     {
-        statList.pop_back();
+        statListPoolAndShp += (sai_serialize_buffer_pool_stat(it) + list_item_delimiter);
```

Comment:
>statListPoolAndShp [](http://example.com/codeflow?start=8&length=18)

It is difficult to read `Shp`. Could you add some comment?

Reply:
How about the name `statListBufferPoolAndSharedHeadroomPool` ?

Reply:
Yes, I can read now.

Reply:
Fixed.